### PR TITLE
More machining recipes

### PR DESCRIPTION
**Pull request by Muncher2112**
Syringe, hatchet, bucket, and beaker recipes.

### CODE REVIEW

**Review by Harcourt1 — COMMENTED** [review]
good.

File: code/modules/crafting/parts/lathe_items.dm [hunk 1]
```diff
@@ -78,10 +78,21 @@
...
-	desc = "This is a piece of glass.  It could be used as a handle"
+	desc = "This is a piece of glass.  It could be used as a handle, or hollowed out into a cup"
 	icon_state = "glass_handle"
 	matter = list("glass" = 400)
 
+	mill(var/mob/user)
+		return /obj/item/glass_handle_hollow
+
+/obj/item/glass_handle_hollow
+	name = "Hollow Glass handle"
```
Comment (HeyBanditoz) [inline code/modules/crafting/parts/lathe_items.dm:89]: "hollow glass handle"

File: code/modules/crafting/recipes/items.dm [hunk 2]
```diff
@@ -23,14 +23,46 @@
...
-	parts = list(/obj/item/cylinder_large_hollow = 1, /obj/item/pipe = 1)
+	parts = list(/obj/item/cylinder_large_hollow = 1, /obj/item/pipe  = 1)
 	tools = list(/obj/item/weapon/weldingtool = 1)
-	result = list(/obj/machinery/portable_atmospherics/canister/empty/air 	 = 1)
+	result = list(/obj/machinery/portable_atmospherics/canister/empty/air = 1)
+	time = 30
+	base_chance = 100
+
+/datum/crafting_recipe/bucket
+	name = "Bucket"
```
Comment (HeyBanditoz) [inline code/modules/crafting/recipes/items.dm:39]: "bucket"

File: code/modules/crafting/recipes/items.dm [hunk 3]
```diff
@@ -23,14 +23,46 @@
...
+	parts = list(/obj/item/cylinder_large_hollow = 1, /obj/item/metal_bar  = 1)
+	tools = list(/obj/item/weapon/weldingtool = 1)
+	result = list(/obj/item/weapon/reagent_containers/glass/bucket = 1)
+	time = 30
+	base_chance = 100
+
+/datum/crafting_recipe/syringe
+	name = "Syringe"
```
Comment (HeyBanditoz) [inline code/modules/crafting/recipes/items.dm:47]: "syringe"

File: code/modules/crafting/recipes/items.dm [hunk 4]
```diff
@@ -23,14 +23,46 @@
...
+	parts = list(/obj/item/metal_shiv = 1, /obj/item/metal_bar  = 1, /obj/item/glass_tube = 1)
+	tools = list()
+	result = list(/obj/item/weapon/reagent_containers/syringe = 1)
+	time = 30
+	base_chance = 100
+
+/datum/crafting_recipe/beaker
+	name = "Beaker"
```
Comment (HeyBanditoz) [inline code/modules/crafting/recipes/items.dm:55]: "beaker"

File: code/modules/crafting/recipes/items.dm [hunk 5]
```diff
@@ -23,14 +23,46 @@
...
+	parts = list(/obj/item/glass_handle_hollow = 1)
+	tools = list(/obj/item/weapon/weldingtool = 1)
+	result = list(/obj/item/weapon/reagent_containers/glass/beaker = 1)
+	time = 30
+	base_chance = 100
+
+/datum/crafting_recipe/hatchet
+	name = "Hatchet"
```
Comment (HeyBanditoz) [inline code/modules/crafting/recipes/items.dm:63]: "hatchet"

**Pull request merged**